<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1,0,0,0,0">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="startStopButton">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="portLabel">
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="portEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="baudrateLabel">
        <property name="text">
         <string>Baudrate:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="baudrateEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="readingIntervalLabel">
        <property name="text">
         <string>Reading interval:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="readingIntervalEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="simulationFileLabel">
        <property name="text">
         <string>Simulation File:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="simulationFileEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCustomPlot" name="plot" native="true"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="massLabel">
        <property name="text">
         <string>Mass:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="massEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="solidDensityLabel">
        <property name="text">
         <string>Solid density:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="solidDensityEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="liquidDensityLabel">
        <property name="text">
         <string>Liquid density:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="liquidDensityEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <property name="topMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="filterMinLabel">
        <property name="text">
         <string>Filter min:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="filterMinEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="filterMaxLabel">
        <property name="text">
         <string>Filter max:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="filterMaxEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="computationIntervalLabel">
        <property name="text">
         <string>Computation interval:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="computationIntervalEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="outputLabel">
      <property name="text">
       <string>No results yet</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="exitButton">
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
